src/pages/Florist/index.tsx

```diff
@@ -113,11 +118,16 @@ export const Florist = () => {
           price: flowerPrice,
         });
       }
+      setFloristStep(0);
```

Comment:
Why are you setting this state here?

Reply:
That's right. Before create `cancelFunction` ,  I thought it is necessary. But now it is not. I will remove this.